internal/logger/target/http/http.go

```diff
@@ -258,7 +258,11 @@ func New(config Config) *Target {
 // Send log message 'e' to http target.
 func (h *Target) Send(entry interface{}) error {
 	if !h.online {
```

Comment:
We need to protect this variable because Send() is called concurrently by many callers, if two callers compete with Init() then they can incorrectly start multiple go-routines that can lead to go-routine leaks.

Reply:
Ack. Let me change the logic.